python/python/glide/async_commands/redis_modules/json.py

```diff
@@ -139,6 +140,98 @@ async def get(
     return cast(str, await client.custom_command(args))
 
 
+async def strappend(
+    client: TRedisClient,
+    key: str,
+    value: str,
+    path: Optional[str] = None,
+) -> TJsonResponse[int]:
+    """
+    Appends the specified `value` to the string stored at the specified `path` within the JSON document stored at `key`.
+
+    See https://redis.io/commands/json.strapped/ for more details.
+
+    Args:
+        client (TRedisClient): The Redis client to execute the command.
+        key (str): The key of the JSON document.
+        value (str): The value to append to the string. Must be wrapped with single quotes.
```

Comment:
> Must be wrapped with single quotes.

since it's not clear if passing 'foo' is enough - please add an example here. 
        value (str): The value to append to the string. Must be wrapped with single quotes. For example, to append "foo", pass '"foo"'.